src/reporters/apiary-reporter.coffee

```diff
@@ -45,10 +46,15 @@ class ApiaryReporter
             ciEnvVar = true
 
         if ciEnvVar == true
-          ciEnvVars[envVarName] = process.env[envVarName]          
-      
+          ciEnvVars[envVarName] = process.env[envVarName]
+
+      # transform blueprints data to array
+      blueprints = []
+      for blueprintPath, blueprintData of blueprintsData
+        blueprints.push blueprintData
```

Comment:
I'd probably add a line with "setup" for the `filename` here too. Something like:
`blueprintData.filename ?= blueprintPath`